doc/howtos/rdtraining/other_module.rst

```diff
@@ -0,0 +1,167 @@
+``estate.property`` model: when a property is sold, we need to add some logic. Does that sound
+familiar? If not, it's a good idea to go back to the
+:ref:`previous chapter <howto/rdtraining/inheritance>` since you might have missed something ;-)
+
+As a first step, we need to extend the action called when pressing the
+:ref:`'Sold' button <howto/rdtraining/actions>` on a property. To do so, we need to create a
+:ref:`model inheritance <howto/rdtraining/inheritance>` in the ``estate_account`` module
+for the ``estate.property`` model. For now, the action overridden will simply return the ``super``
+call. Maybe an example will make things clearer::
+
+    from odoo import models
+
+    class InheritedModel(models.Model):
+        _inherit = "inherited.model"
+
+        def inherited_action(self):
+            return super().inherited_action()
+
+A practical example can be found
+`here <https://github.com/odoo/odoo/blob/f1f48cdaab3dd7847e8546ad9887f24a9e2ed4c1/addons/event_sale/models/account_move.py#L7-L16>`__.
+
+.. exercise:: First step of invoice creation
+
+    - Create a ``estate.property.py`` file in the right folder of the ``estate_account`` module.
+    - ``_inherit`` the ``estate.property`` model
+    - Override ``action_sold`` method (you might have named it differently) to return the ``super``
+      call.
+
+    Tip: to make sure it works, add a ``print`` or a debugger breakpoint in the overridden method.
+
+Is it working? If not, maybe check all Python files are correctly imported.
+
+When the override is working, we can move forward and create the invoice. Unfortunately, there
+is no easy way to know how to create any given object in Odoo. Most of the time, it is necessary
+to have a look at its model to find the required fields and provide appropriate values.
+
+A good way to learn is to look how other modules do what you want to do. For example, one of
+the basic flows of Sales is the creation of an invoice from a sales order. That looks like a good
+starting point since it does exactly what we want to do. Take some time to read and understand the
+`_create_invoices <https://github.com/odoo/odoo/blob/f1f48cdaab3dd7847e8546ad9887f24a9e2ed4c1/addons/sale/models/sale.py#L610-L717>`__
+method. When you are done crying because this simple task looks awfully complex, we can move
+forward on the tutorial.
+
+To create an invoice, we need the following information:
+
+- a ``partner_id``: the customer
+- a ``move_type`` which has several `possible values <https://github.com/odoo/odoo/blob/f1f48cdaab3dd7847e8546ad9887f24a9e2ed4c1/addons/account/models/account_move.py#L138-L147>`__
+- a ``journal_id``: the accounting journal
+
+This is enough to create an empty invoice.
+
+.. exercise:: Second step of invoice creation
+
+    Create an empty ``account.move`` in the override of the ``action_sold`` method:
+
+    - the ``partner_id`` is taken from the current ``estate.property``
```

Comment:
Maybe we need to explain that for the Many2one, you only give the id and not the object.